integration-tests/api/__tests__/store/product-category.ts

```diff
@@ -1,358 +1,368 @@
-import {ProductCategory} from "@medusajs/medusa"
-import path from "path"
-
-import startServerWithEnvironment
-  from "../../../environment-helpers/start-server-with-environment"
-import {useApi} from "../../../environment-helpers/use-api"
-import {useDb} from "../../../environment-helpers/use-db"
-import {simpleProductCategoryFactory} from "../../../factories"
+import { medusaIntegrationTestRunner } from "medusa-test-utils"
+import {
+  adminHeaders,
+  createAdminUser,
+} from "../../../helpers/create-admin-user"
+import { breaking } from "../../../helpers/breaking"
 
 jest.setTimeout(30000)
 
-describe("/store/product-categories", () => {
-  let medusaProcess
-  let dbConnection
-  let productCategory!: ProductCategory
-  let productCategory2!: ProductCategory
-  let productCategoryChild!: ProductCategory
-  let productCategoryParent!: ProductCategory
-  let productCategoryChild2!: ProductCategory
-  let productCategoryChild3!: ProductCategory
-  let productCategoryChild4!: ProductCategory
-
-  beforeAll(async () => {
-    const cwd = path.resolve(path.join(__dirname, "..", ".."))
-    const [process, connection] = await startServerWithEnvironment({
-      cwd,
-      env: { MEDUSA_FF_PRODUCT_CATEGORIES: true },
-    })
-    dbConnection = connection
-    medusaProcess = process
-  })
-
-  afterAll(async () => {
-    const db = useDb()
-    await db.shutdown()
-
-    medusaProcess.kill()
-  })
-
-  beforeEach(async () => {
-    productCategoryParent = await simpleProductCategoryFactory(dbConnection, {
-      name: "category parent",
-      description: "test description",
-      is_active: true,
-      is_internal: false,
-      rank: 0,
-    })
-
-    productCategory = await simpleProductCategoryFactory(dbConnection, {
-      name: "category",
-      parent_category: productCategoryParent,
-      is_active: true,
-      rank: 0,
-    })
-
-    productCategoryChild = await simpleProductCategoryFactory(dbConnection, {
-      name: "category child",
-      parent_category: productCategory,
-      is_active: true,
-      is_internal: false,
-      rank: 3,
-    })
-
-    productCategoryChild2 = await simpleProductCategoryFactory(dbConnection, {
-      name: "category child 2",
-      parent_category: productCategory,
-      is_internal: true,
-      is_active: true,
-      rank: 0,
+medusaIntegrationTestRunner({
```

Comment:
There are still some tests that are failing, but I think we have some bugs in how product categories are handled. I'll investigate that separately, I want to focus on getting the storefront running first.